presto-main/src/main/java/com/facebook/presto/cost/TableScanStatsRule.java

```diff
@@ -80,14 +80,15 @@ public TableScanStatsRule(Metadata metadata, StatsNormalizer normalizer)
 
     private SymbolStatsEstimate toSymbolStatistics(TableStatistics tableStatistics, ColumnStatistics columnStatistics, Session session, Type type)
     {
+        double nullsFraction = columnStatistics.getNullsFraction().getValue()
+                / (columnStatistics.getNullsFraction().getValue() + columnStatistics.getOnlyRangeColumnStatistics().getFraction().getValue());
```

Comment:
`columnStatistics.getNullsFraction().getValue() + columnStatistics.getOnlyRangeColumnStatistics().getFraction().getValue()`

Isn't the above should sum to `1`?